source/concepts/what-is-ember-js.md

```diff
@@ -124,15 +124,18 @@ You can tell Ember about these dependencies like this:
 
 ```javascript
 var president = Ember.Object.create({
-  firstName: "Barack",
-  lastName: "Obama",
+  init: function() {
+    this._super.apply(this, arguments);
+    this.set('firstName', 'Barack');
```

Comment:
the alternative I linked to above now shows the ideal behavior as @thejameskyle describes. Not sure if I read the output incorrectly or if there was a deploy of the repl.

**So,** using `...` at all times seems reasonable to me actually.